gradlew

```diff
@@ -84,6 +84,10 @@ Please set the JAVA_HOME variable in your environment to match the
 location of your Java installation."
 fi
 
+# Create the neccessary templates for IPython
+mkdir ~/.ipython/kernels/kotlin
+touch ~/.ipython/kernels/kotlin/kernel.json
```

Comment:
And there is no need to touch it since the build.gradle script will create it upon writing.